README.md

```diff
@@ -1,6 +1,9 @@
 # TSLGen
 Template Vector Library Generator
```

Comment:
Template Simd Library (formerly known as Template Vector Library (TVL*))...




* As we are database people, we frequently experienced confusion regarding the term Vector (since it is predominately associated with batched processing). To prevent this confusion, we decided to rename our library.